pkg/scheduler/algorithm/scheduler_interface.go

```diff
@@ -84,8 +85,9 @@ type ScheduleAlgorithm interface {
 	Preempt(*v1.Pod, NodeLister, error) (selectedNode *v1.Node, preemptedPods []*v1.Pod, cleanupNominatedPods []*v1.Pod, err error)
 	// Predicates() returns a pointer to a map of predicate functions. This is
 	// exposed for testing.
-	Predicates() map[string]FitPredicate
+	Predicates() map[string]predicates.FitPredicate
```

Comment:
@misterikkit  There is an import loop because pkg/scheduler/algorithm import pkg/scheduler/algorithm/predicates.